### ISSUE TITLE
Multiple PRs? (Yes!)

### ISSUE THREAD

**Issue by Priultimus** [wzhouwzhou/one-line-wonders#8]
Can I submit multiple PRs or no?

**Comment by kannagikazuko** [comment]
Yes! Multiple PRs are welcome and each individual PR will count separately towards your 5 PRs for a free Hacktober shirt! Please make sure each PR is documented for all changes!

**Comment by Priultimus** [comment]
Oh okay, thanks!

**Comment by kannagikazuko** [comment]
I re-opened the issue in case new contributors are confused about this.
